webapp/src/plugin_constants/index.ts

```diff
@@ -46,6 +46,7 @@ const IncidentCreatedMsg = 'Incident created successfully!';
 const ChannelPanelToggleLabel = 'Subscribe to the new incident';
 const MaxShortDescriptionCharactersView = 75;
 const MaxShortDescriptionLimit = 160;
+const SiteUrl = 'SiteUrl';
```

Comment:
```suggestion
const SiteUrl = 'SITEURL';
```
Mattermost generally has its cookies in capital case.